opencontrail/cassandra_backup.sls

```diff
@@ -0,0 +1,7 @@
+/usr/local/bin/contrail-cassandra-backup:
```

Comment:
I would prefer not introducing new state (cassandra_backup.sls) but having this as a part of opencontrail.control